content/en/docs/guides/air-gapped-gitops-with-ocm-and-flux.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "Air-gapped GitOps with OCM & Flux"
+description: "Air-gapped GitOps with OCM & Flux"
+lead: ""
+date: 2022-11-23T10:00:00+00:00
+lastmod: 2022-11-23:00:00+00:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: ""
+    identifier: "air-gapped-gitops-with-ocm-flux"
+weight: 103
+toc: true
+---
+
+## Introduction
+
+In this guide we will show how the tools provided by OCM that make it possible to automate your air-gapped deployments.
+
+Air-gapped can mean different things depending on the context. For the purposes of this guide we'll assume it means your deployment artifacts are stored in a private registry that is protected by the security controls put in place at your organization. Your applications only have access to this private registry and have little to no public internet access.
+
+We'll take the same `podinfo` component that we deployed in the [Deploy Applications with OCM & GitOps](/docs/guides/deploying-applications-with-gitops) guide but this time we shall use the OCM CLI to transfer the component to our own registry. The application will then be deployed from this "private" registry. This, of course, but mimics a real world air-gap scenario. In practice there could be many layers of security between the two registries; however the mechanics are ultimately the same.
+
+## Table of contents
+- [Requirements](#requirements)
+- [Component Content](#component-content)
+- [Component Transfer](#component-transfer)
+- [Gitops & Localization](#gitops-localization)
+- [Verification](#verification)
+- [To Be Continued](#to-be-continued)
+- [Conclusion](#to-be-continued)
+
+## Requirements
+
+- [OCM command line tool](https://github.com/open-component-model/ocm)
+- [kubectl](https://kubernetes.io/docs/reference/kubectl/)
+- [git](https://git-scm.com/downloads)
+- [gh](https://github.com/cli/cli)
+- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+- [flux](https://fluxcd.io/flux/installation/#install-the-flux-cli)
+
+### Component Content
+
+The `podinfo` component contains three resources:
+- a container image for podinfo
+- a kubernetes deployment manifest for podinfo
+- a configuration file read by the ocm-controller
+
+We can list these resources using the `ocm` CLI:
+
+```bash
+ocm get resources ghcr.io/phoban01//phoban.io/podinfo -c v6.3.5
```

Comment:
```suggestion
ocm get resources ghcr.io/phoban01/phoban.io/podinfo -c v6.3.5
```

Reply:
Double-slash is actually the correct syntax ... part before is the registry, part after is the component name.

Reply:
Oh ops. Sorry about that.